servers/zts/src/main/java/com/yahoo/athenz/zts/cert/CertRecordStoreConnection.java

```diff
@@ -70,4 +71,20 @@
      * @return number of records deleted
      */
     int deleteExpiredX509CertRecords(int expiryTimeMins);
+
+    /**
```

Comment:
Similar to the logic of updateRoleMemberExpirationNotificationTimestamp and getNotifyTemporaryRoleMembers in ZMS.